jpirumvaa/jpirumva
# #4VKc6q9X/13 Update Existing Article

#### What does this PR do?
Once a user clicks the edit option, he has to see a form that allows him to update information. He has to update an article and once he hits submit, the article should be updated.
#### Description of Task to be completed?
Have an update form UI, see update form once a user clicks the update icon, see changes one update button is clicked on the form. 
#### How should this be manually tested?
To test this, clone the branch, run the app, and see if these options are working:
1. Clicking on edit icon gives edit form with current values
2. Clicking update article button on form alerts either updated successfully or an error occurred.
3. After submitting an updated article, the updates should be seen. If the article's title was edited, it will reflect on the admin's page immediately.
4. The updated blog should appear updated in Blogs page
#### Any background context you want to provide?
Since I have not implemented a continue reading option, you may find blog details that don't change.
#### What are the relevant pivotal tracker stories?
[https://trello.com/c/nc7M9IeC/46-update-existing-article]()
#### Screenshots (if appropriate)
![update](https://user-images.githubusercontent.com/50476344/89727879-e5f7f100-da28-11ea-9027-290af4f459a2.png)
